src/explore-education-statistics-admin/src/pages/release/datablocks/components/__tests__/DataBlockDetailsForm.test.tsx

```diff
@@ -77,6 +77,37 @@ describe('DataBlockDetailsForm', () => {
     });
   });
 
+  test('shows validation error if has whitespace featured table name', async () => {
```

Comment:
Think this test is a bit redundant since no error will show for this scenario

Reply:
How do you mean? I think after my changes, if you put a bunch of spaces in the featured table name and try to save it, it'll throw the error "Enter a featured table name".

Reply:
Why would we want this to be the case? We shouldn't bother the user with this and just trim the string before sending it to the server.

Reply:
I've only performed the trim on `highlightName` and `highlightDescription` - the other fields still have validation issues, but I think that's outside the remit of the work I'm doing.

I'll create a ticket to test inputs across the service to see what mischief could be achieved, and then we can fix them all at once if that's a priority. The reason I didn't was because I think this is a problem across the codebase I'm not sure how big of a priority this is, as analysts are unlikely to sabotage their own releases in this way. But to be honest, that's for Cam to decide, so I'll create the ticket.

Reply:
Ah right ok I see, good shout. This wasn't quite the point I was trying to get across in this original comment however. My main point was that we're trimming the string before performing any validation that will surface to the end user. i.e. if we had `Yup.string().trim()` the only validation that will be performed is that the input must be of type `string` (and we trim it after the fact). No error will show up around a string containing whitespace

Reply:
You mean like if someone input a string like `"    Featured table name    "` which would automatically be trimmed to `"Featured table name"`?

I think in this scenario we should just trim the string and don't need to provide a validation error. I don't think there is a scenario where a user would want whitespace before or after their featured table name, and if we did throw a validation error on strings like this, it just means users will have to trim the whitespace manually. So it makes sense to me to just trim it automatically.

Reply:
Maybe the confusion is around the name of the unit test you're querying? That test was supposed to cover featured table names that are entirely whitespace - i.e. no characters. The second unit test is to cover the situation where there are leading and/or trailing whitespace. 

I guess you could argue that the validation error should be different based on whether the input name is empty or full of whitespace, but that didn't seem worth the effort to me.

Reply:
Yep that's right, if someone enters a string with loads of whitespace it'll be automatically trimmed to "Featured table name" without surfacing a validation error. This is what we're doing at the moment in this PR (and across the codebase atm). I see what the test is trying to do now, I think the test title is a bit misleading. Maybe it could be renamed to something such as `shows validation error if the featured table name is entirely a whitespace string` or something along those lines